app/assets/stylesheets/custom.scss

```
@@ -587,5 +587,11 @@ table {
     h2 {
       color: $brand;
     }
+
+    &.budget-heading-background {
+      background: $brand image-url("background-header-budget.jpg"); //pending  add image from zepling
```

```suggestion
      background: #333 image-url("background-header-budget.jpg"); //pending  add image from zepling
```

This way there'll be no pink background color while the image is loaded. The pink background would mean the "button" would be invisible :thinking:.

---

Sorry, the previous comment was meant for the homepage: `background: #333 image-url("background-header-budget.jpg"); //pending  add image from zepling`

---

> pending  add image from zepling

It looks like this comment isn't accurate for budgets, since it's actually the image used in Zeplin :thinking:.